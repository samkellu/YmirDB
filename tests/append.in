append a 1 2
set a 123 33 554
set b 11 a 46 22
get a
push a b
set c 123 334 b
append a 44 22 c
get a
bye
