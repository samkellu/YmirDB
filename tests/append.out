> no such key

> ok

> ok

> [123 33 554]

> ok

> ok

> ok

> [b 123 33 554 44 22 c]

> bye
